app/views/thread_mailer/new_vote.html.haml

```diff
@@ -1,7 +1,2 @@
-= render '/application/mailer_doctype'
-%html
-  %head
-    = stylesheet_link_tag 'email'
-  %body
-    = render "vote", vote: @vote
-    = render "footer", link: motion_url(@motion, @utm_hash)
+= render "vote", vote: @vote
+= render "footer", link: motion_url(@motion, @utm_hash)
```

Comment:
I'm wondering if we could pull out the footer into the layout as well, and render it iff we provide a link in the action?

```
def new_motion
  ...
  @footer_link = motion_url(@motion, @utm_hash)
end
```
```
render 'footer', link: @footer_link if @footer_link
```
💛